dotnet/src/SemanticKernel/Memory/Collections/Score.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.SemanticKernel.Memory.Collections;
+
+/// <summary>
+/// Structure for storing score value.
+/// </summary>
+internal struct Score : IComparable<Score>, IEquatable<Score>
+{
+    public double Value { get; set; }
+
+    public static Score Min => double.MinValue;
```

Comment:
Are these used anywhere?

Reply:
`Min` property is used to initialize `MinHeap` data structure with initial score. `Max` property would be used in the same way for `MaxHeap`, but since `MaxHeap` doesn't exist yet, I will removed this property for now.